mini_pupper_control/mini_pupper_control/vel_to_servo_controller.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+#
+# Copyright 2023 MangDang
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This program is based on
+# https://github.com/stanfordroboticsclub/StanfordQuadruped/blob/master/run_robot.py
+# which is released under the MIT License.
+# Copyright (c) 2020 Stanford Student Robotics
+
+import numpy as np
+# from .StanfordQuadruped.src.IMU import IMU
+from .StanfordQuadruped.src.Controller import Controller
+from .StanfordQuadruped.src.Command import Command
+from .StanfordQuadruped.src.State import State
+from .StanfordQuadruped.pupper.Kinematics import four_legs_inverse_kinematics
+try:
+    from MangDang.mini_pupper.Config import Configuration
+except ImportError:
+    from .StanfordQuadruped.pupper.Config import Configuration
```

Comment:
```suggestion
from MangDang.mini_pupper.Config import Configuration
```

Since mini_pupper_bsp or mini_pupper_2_bsp is required, we can say it is OK if it returns an error if it cannot be imported.

Reply:
To ensure compatibility with the simulation environment, it is recommended that vel_to_servo_controller should not ONLY rely on the hardware bsp package. It would be a good choice to include an environment check in the import statements.

Reply:
How can I test that this `from .StanfordQuadruped.pupper.Config import Configuration` is necessary in a simulation environment?

Reply:
the colleague who is responsible for the simulation on the Mangdang team has indicated that he is considering using the vel_to_servo_controller, and he might be able to provide more detailed information regarding this matter. @0nhc